src/chain-spec.ts

```diff
@@ -210,6 +217,28 @@ export async function addAuraAuthority(
   );
 }
 
+export async function addGrandpaAuthority(
+  specPath: string,
+  name: string,
+  accounts: any,
+) {
+  const { ed_account } = accounts;
+
+  const chainSpec = readAndParseChainSpec(specPath);
+
+  let keys = getAuthorityKeys(chainSpec, "grandpa");
+  if (!keys) return;
+
+  keys.push([ed_account.address, 1]);
+
+  writeChainSpec(specPath, chainSpec);
+  console.log(
+    `\t👤 Added Genesis Authority (GRANDPA) ${decorators.green(
+      name,
+    )} - ${decorators.magenta(ed_account.address)}`,
+  );
```

Comment:
Maybe use the [logtable](https://github.com/paritytech/zombienet/blob/main/src/utils/logger.ts#L56) instead of the console log?

Reply:
I'm not sure if we want to use `logtable` for all logs, but we can chat about it 👍

Reply:
Not a priortiy 1 at the moment. So lets not worry about it